Arista/Ticks/disconnects.tick

```diff
@@ -1,7 +1,7 @@
 var UPDATES_IN_DISCONNECT_CHECK = 240
 var UPDATES_IN_ONE_DISCONNECT = 40
 var DISCONNECTS_CHECK_PERIOD = 1h
-var HOST_CHECK_PERIOD = 1m
+var HOST_CHECK_PERIOD = 15m
```

Comment:
The constants are currently named same but these are per script constants.

For example - its possible the that disconnect tick script is made much faster than hostdown script etc. IMHO its better to keep these constants close to the Tick scripts that use/depend on them.

Also there is no #include kind of directive in the tick script for including another file